### DISCUSSION TITLE
No indication on search bar

### DISCUSSION THREAD

**Discussion by premdevai** [mdn/mdn-community#134] [Browser compatibility data]
![image](https://user-images.githubusercontent.com/40886278/174536832-38a44dbf-6560-492e-b47e-f0a1700e54db.png)

as you can learly see that there is no clear indication wether i  clicked on the search input fiels or not 
please give a feed back to the user

**Comment by schalkneethling** [comment]
Hey @premshetty, thank you for bringing this to our attention. It sounds to me like this is a user experience problem that is well enough defined to be an issue. Would you mind opening and issue here? https://github.com/mdn/yari/issues/new/choose

**Reply by cw118** [reply]
I believe mdn/yari#6584 was also opened to report this :)

**Comment by pepelsbey** [comment]
It seems like the [bug has been filed](https://github.com/mdn/yari/issues/6584), and this discussion is not relevant anymore. Closing as resolved.
